core/src/main/scala/org/apache/spark/rdd/MapPartitionsRDD.scala

```diff
@@ -32,12 +32,16 @@ import org.apache.spark.{Partition, TaskContext}
  *                              doesn't modify the keys.
  * @param isFromBarrier Indicates whether this RDD is transformed from an RDDBarrier, a stage
  *                      containing at least one RDDBarrier shall be turned into a barrier stage.
+ * @param orderSensitiveFunc whether or not the function is order-sensitive. If it's order
+ *                           sensitive, it may return totally different result if the input order
```

Comment:
`it may return totally different result if the input order changed ` -> `it may return totally different result when the input order is changed`